doc/man3/SSL_CTX_set_stateless_cookie_generate_cb.pod

```diff
@@ -0,0 +1,62 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_stateless_cookie_generate_cb,
+SSL_CTX_set_stateless_cookie_verify_cb
+- Callback functions for stateless TLS1.3 cookies
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void SSL_CTX_set_stateless_cookie_generate_cb(
+     SSL_CTX *ctx,
+     int (*gen_stateless_cookie_cb) (SSL *ssl,
+                               unsigned char
+                               *cookie,
+                               size_t
+                               *cookie_len));
+ void SSL_CTX_set_stateless_cookie_verify_cb(
+     SSL_CTX *ctx,
+     int (*verify_stateless_cookie_cb) (SSL *ssl,
+                                  const unsigned char
+                                  *cookie,
+                                  size_t
+                                  cookie_len));
```

Comment:
How did we choose the indentation level for these arguments?  (It doesn't look wrong per se, it just seems pretty arbitrary.)

Reply:
Fell out of alignment when the argument was renamed. Thanks for the catch, fixing.